git/git_remotes.md

```diff
@@ -1,22 +1,70 @@
 # Git Remotes
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+In this lesson, we will cover the `git remote` command and remote repositories. 
+
+A remote is like a bookmark that tells Git where the other versions of a project are stored. These versions are housed in "remote repositories." 
+
+Remote repositories are the versions of your project that are not on your local machine. These are typically the repositories that you are pulling code from and pushing code to when you use Git.
 
 ## Learning outcomes
 *Look through these now and then use them to test yourself after doing the assignment*
 
-what the student is expected to know or be able to do by the end of this lesson
+By the end of this you should be able to:
+
+* Explain why we use remote repositories
+* Fork and clone a repository to your machine
+* Add a remote to a repository
+* View remotes on a local project
+* Remove a remote from a repository
+* Pull changes from a remote branch
+* Push changes to a remote branch
+* Submit a pull request to a remote repository that doesn't belong to you
+
+# Introduction to Remotes
+This section will give you a clear understanding of what a remote is and why we use remotes. 
```

Comment:
We can remove this sentence, the title of the section should be enough to communicate what the section is about 😄